sklearn/metrics/_plot/cap_curve.py

```diff
@@ -0,0 +1,274 @@
+    a :class:`~sklearn.metrics.CumulativeAccuracyDisplay`. All parameters are
+    stored as attributes.
+
+    Read more in the :ref:`User Guide <visualizations>`.
+
+    Parameters
+    ----------
+    cumulative_true_positives : ndarray
+        Cumulative number of true positives.
+
+    cumulative_total : ndarray
+        Cumulative number of cases examined.
+
+    estimator_name : str, default=None
+        Name of estimator. If None, the estimator name is not shown.
+
+    pos_label : int, float, bool or str, default=None
+        The class considered as the positive class when computing the metrics.
+        By default, `estimators.classes_[1]` is considered as the positive class.
+
+    Attributes
+    ----------
+    line_ : matplotlib Artist
+        CAP Curve.
+
+    ax_ : matplotlib Axes
+        Axes with CAP Curve.
+
+    figure_ : matplotlib Figure
+        Figure containing the curve.
+    """
+
+    def __init__(self, *, cumulative_true_positives, cumulative_total, estimator_name=None, pos_label=None):
+        import numpy as np
+        self.estimator_name = estimator_name
+        self.cumulative_true_positives = cumulative_true_positives
+        self.cumulative_total = cumulative_total
+        self.pos_label = pos_label
+
+    def plot(
+        self,
+        ax=None,
+        *,
+        name=None,
+        **kwargs,
+    ):
+        """Plot visualization.
+
+        Extra keyword arguments will be passed to matplotlib's ``plot``.
+
+        Parameters
+        ----------
+        ax : matplotlib axes, default=None
+            Axes object to plot on. If `None`, a new figure and axes is created.
+
+        name : str, default=None
+            Name of CAP Curve for labeling. If `None`, use `estimator_name` if
+            not `None`, otherwise no labeling is shown.
+
+        **kwargs : dict
+            Keyword arguments to be passed to matplotlib's `plot`.
+
+        Returns
+        -------
+        display : :class:`~sklearn.metrics.CumulativeAccuracyDisplay`
+            Object that stores computed values.
+        """
+        self.ax_, self.figure_, name = self._validate_plot_params(ax=ax, name=name)
+
+        line_kwargs = {"label": name} if name is not None else {}
+        line_kwargs.update(**kwargs)
+
+        (self.line_,) = self.ax_.plot(self.cumulative_total, self.cumulative_true_positives, **line_kwargs)
+
+        xlabel = "Total Cases Examined"
+        ylabel = "Cumulative True Positives"
+        self.ax_.set(xlabel=xlabel, ylabel=ylabel, aspect="equal")
+
+        if "label" in line_kwargs:
+            self.ax_.legend(loc="lower right")
+
+        return self
+
+    @classmethod
+    def from_predictions(
+        cls,
+        y_true,
+        y_pred,
+        *,
+        sample_weight=None,
+        pos_label=None,
+        name=None,
+        ax=None,
+        **kwargs,
+    ):
+        """Plot CAP curve given the true and predicted values.
+
```

Comment:
Please also mention the other names for this curve so that people can google it from any name.